historical_surveillance/migrations/0001_initial.py

```diff
@@ -0,0 +1,74 @@
+# Generated by Django 3.1.7 on 2021-04-04 17:21
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
```

Comment:
When I run 
`./manage.py showmigrations`
or any ./manage.py command I'm getting this error:

>     _mysql.connection.query(self, query)
django.db.utils.ProgrammingError: (1146, "Table 'lucia_education.historical_surveillance_district' doesn't exist")

Just me?

Reply:
Yes. I will correct that. It is attempting to get values from the tables before actually creating it.